bot/cogs/doc/cog.py

```diff
@@ -126,6 +79,7 @@ class InventoryURL(commands.Converter):
     @staticmethod
     async def convert(ctx: commands.Context, url: str) -> str:
         """Convert url to Intersphinx inventory URL."""
+        await ctx.trigger_typing()
         try:
             intersphinx.fetch_inventory(SPHINX_MOCK_APP, '', url)
```

Comment:
Thoughts on making this async by running it an an executor? I know it uses `requests` internally.

Reply:
Good catch, sound like a good idea, didn't look at the code here much as the second fetch already uses an executor.

I was thinking about making an issue from the comment I made in #dev-contrib on intersphinx, where we could ditch the dependency and adapt the code to be fully async which would also solve this here without an executor